src/engine/audio/sound.cpp

```diff
@@ -126,47 +68,26 @@ void sound_update()
 	position.y = -camera.z / 32;
 	position.z = -camera.x;
 
-	/*double cosroll = cos(0);
-	double cosyaw = cos(camera.ang);
-	double cospitch = cos(camera.vang);
-	double sinroll = sin(0);
-	double sinyaw = sin(camera.ang);
-	double sinpitch = sin(camera.vang);
-
-	double rx = sinroll*sinyaw - cosroll*sinpitch*cosyaw;
-	double ry = sinroll*cosyaw + cosroll*sinpitch*sinyaw;
-	double rz = cosroll*cospitch;*/
-
 	forward.x = 1 * sin(camera.ang);
 	forward.y = 0;
 	forward.z = 1 * cos(camera.ang);
-	/*forward.x = rx;
-	forward.y = ry;
-	forward.z = rz;*/
-
-	/*rx = sinroll*sinyaw - cosroll*cospitch*cosyaw;
-	ry = sinroll*cosyaw + cosroll*cospitch*sinyaw;
-	rz = cosroll*sinpitch;*/
 
 	up.x = 0;
 	up.y = 1;
 	up.z = 0;
-	/*up.x = rx;
-	up.y = ry;
-	up.z = rz;*/
 
 	//FMOD_System_Set3DListenerAttributes(fmod_system, 0, &position, &velocity, &forward, &up);
-	FMOD_System_Set3DListenerAttributes(fmod_system, 0, &position, 0, &forward, &up);
+	fmod_system->set3DListenerAttributes(0, &position, 0, &forward, &up); //TODO: For splitscreen, change the listener number to the current player...
```

Comment:
@SheridanR FYI

Reply:
Thanks for pointing this out, I made a personal note about it